plotly/plotlyfig.m

```diff
@@ -946,7 +946,7 @@ function delete(obj)
                             strcmpi(fieldname,'surface') || strcmpi(fieldname,'scatter3d') ...
                         ||  strcmpi(fieldname,'mesh3d') || strcmpi(fieldname,'bar') ...
                         ||  strcmpi(fieldname,'scatterpolar') || strcmpi(fieldname,'barpolar') ...
-                        )
+                        || strcmpi(fieldname,'scene'))
```

Comment:
the struct scene should be added in the file plotly_reference.mat or alternatively this line of code